contracts/HydraStaking/modules/VestedStaking/VestedStaking.sol

```diff
@@ -40,7 +40,8 @@ abstract contract VestedStaking is IVestedStaking, Staking, Vesting {
             end: block.timestamp + duration,
             base: aprCalculatorContract.getBaseAPR(),
             vestBonus: aprCalculatorContract.getVestingBonus(durationWeeks),
-            rsiBonus: uint248(aprCalculatorContract.getRSIBonus())
+            rsiBonus: uint248(aprCalculatorContract.getRSIBonus()),
+            commission: 0
```

Comment:
you can miss that line

Reply:
the compiler exclusively tells me to make sure all parts of the Struct are defined on initialization, even if they are 0